sepolicy/perspectived.te

```diff
@@ -26,7 +26,7 @@ binder_service(perspectived)
 
 # register Binder service
 allow perspectived mperspective_service:service_manager add;
-allow perspectived default_android_service:service_manager add;
+allow perspectived default_android_service_maru:service_manager add;
```

Comment:
On my Pixel 3a and 3a XL, we can let `perspectived` add `default_android_service` to `service_manager` with commit https://github.com/maruos/vendor_maruos/commit/47d1288b89c07c6094fded25e5bdc9a9544005a9. Does it not work on your device?

Reply:
commit 47d1288 does not work for my device. I had to remove lines 518 and 520 in order to get the build to pass the sepolicycheck. Im not sure if this effects other devices or not.
> 518 neverallow * default_android_service:service_manager add;
   519 neverallow * default_android_vndservice:service_manager { add find };
   519 neverallow * default_android_vndservice:service_manager { add find };

The code has this insert:
> Do not allow service_manager add for default service labels.
   Instead domains should use a more specific type such as
   system_app_service rather than the generic type.
   New service_types are defined in {,hw,vnd}service.te and new mappings
   New service_types are defined in {,hw,vnd}service.te and new mappings

This change tried to keep to the spirit of this advice. I created a new service_type specific to maru.

I think this should apply the exact same permissions, just using a maru specific label instead of the default one. So I think this should be safe for other devices as well.

On a different note, it may make sense to rename these new service_types to  maru_android_service and maru_android_hwservice if we decide to go with this.